main.go

```diff
@@ -93,6 +94,10 @@ func main() {
 		"deployments-namespace",
 		"",
 		"The namespace where the kubewarden resources will be created.")
+	flag.StringVar(&controllerWebhookServiceName,
+		"controller-webhook-service-name",
+		"",
+		"The controller webhook service name")
```

Comment:
Do we really need it? I think this was needed in the past before we used Tilt for local development.